api-dummy.sha1

```diff
@@ -1 +1 @@
-9a033bd285b9e74b08b3c01a1e502f24313ec1b2
+e42f9ffac86311dcaa5bd28f878e7fefce934330
```

Comment:
This `api-dummy.sha1` file is probably unused now as we don't even checkout this Git repository in https://github.com/elifesciences/journal-formula/blob/master/salt/journal/init.sls but only use the Docker image. I'll make a follow-up PR to remove it.